tabs/tab_logit.py

```
@@ -363,64 +433,85 @@ def _run_logit():
             fig_crude = None
             
-            if aor_res:
-                df_adj = pd.DataFrame([{'variable': k, **v} for k, v in aor_res.items()])
-                if not df_adj.empty:
-                    fig_adj = create_forest_plot(
-                        df_adj, 'aor', 'ci_low', 'ci_high', 'variable', pval_col='p_value',
-                        title="<b>Multivariable: Adjusted OR</b>", x_label="Adjusted OR"
-                    )
+            x_label = "Adjusted IRR" if model_type == 'poisson' else "Adjusted OR"
+            x_label_crude = "Crude IRR" if model_type == 'poisson' else "Crude OR"
+            col_key = 'aor' # The key in dict is still 'aor'/'or' from logic for consistency
```

Ruff (0.14.10)

Local variable `col_key` is assigned to but never used

Remove assignment to unused variable `col_key`

(F841)